src/drivers/Hrs3300.cpp

```diff
@@ -26,18 +28,21 @@ void Hrs3300::Init() {
   vTaskDelay(100);
 
   // HRS disabled, 12.5 ms wait time between cycles, (partly) 20mA drive
-  WriteRegister(static_cast<uint8_t>(Registers::Enable), 0x60);
+  // Wait time changed to zero and current to 12.5mA - Ceimour
+  WriteRegister(static_cast<uint8_t>(Registers::Enable), 0x70);
 
   // (partly) 20mA drive, power on, "magic" (datasheet says both
   // "reserved" and "set low nibble to 8" but 0xe gives better results
   // and is used by at least two other HRS3300 drivers
-  WriteRegister(static_cast<uint8_t>(Registers::PDriver), 0x6E);
+  // Current 12.5mA and more magic (low nibble 0xF) - Ceimour
+  WriteRegister(static_cast<uint8_t>(Registers::PDriver), 0x2f);
```

Comment:
Old comments don't need to be retained. Clearly explain what the magic values do. Maybe make them binary.

Reply:
Agreed, and you don't need to document the changes in the code, git does that for you.

Reply:
Old comments removed and current comments updated.
Left the driver constants as hex for now. Let me know if you feel strongly about binary constants and I'll change them. Historically they have been hex in most other available example code.

Reply:
Old comments are still retained, so there are contradictory comments.